internal/infrastructure/kubernetes/proxy/resource.go

```diff
@@ -132,6 +134,7 @@ func expectedProxyContainers(infra *ir.ProxyInfra,
 			TrustedCA:   filepath.Join("/sds", common.SdsCAFilename),
 		},
 		MaxHeapSizeBytes: maxHeapSizeBytes,
+		XdsServerHost:    ptr.To(fmt.Sprintf("envoy-gateway.%s.svc.%s", namespace, dnsDomain)),
```

Comment:
can we use `config.EnvoyGatewayServiceName` here ?

Reply:
done